include/vicon_estimator.h

```diff
@@ -33,6 +33,9 @@ static const double kDefaultDt = 0.01;
 // values
 static const double kDefaultTranslationalKp = 1.0;
 static const double kDefaultTranslationalKv = 10.0;
+static const double kDefaultOutlierThresholdM = 0.5;
```

Comment:
What does this trailing M mean? Meters? In general I would agree that having the units there is a good idea. However in this package this is not standard. For consistency I would leave this out.